client/my-sites/earn/memberships/index.jsx

```diff
@@ -237,6 +252,49 @@ class MembershipsSection extends Component {
 							.join( ', ' ) }
 					</div>
 				</CompactCard>
+				<CompactCard
+					onClick={ () =>
+						this.setState( { disconnectedConnectedAccountId: this.props.connectedAccountId } )
+					}
+					className="memberships__settings-link"
+				>
+					<div className="memberships__settings-content">
+						<p className="memberships__settings-section-title is-warning">
+							{ this.props.translate( 'Disconnect Stripe Account' ) }
+						</p>
+						<p className="memberships__settings-section-desc">
+							{ this.props.translate( 'Disconnect Recurring Payments from your Stripe account' ) }
+						</p>
+					</div>
+				</CompactCard>
+				<Dialog
+					isVisible={ !! this.state.disconnectedConnectedAccountId }
+					buttons={ [
+						{
+							label: this.props.translate( 'Cancel' ),
+							action: 'cancel',
+						},
+						{
+							label: this.props.translate( 'Disconnect Recurring Payments from Stripe' ),
+							isPrimary: true,
+							action: 'disconnect',
+						},
+					] }
+					onClose={ this.onCloseDisconnectStripeAccount }
+				>
+					<h1>{ this.props.translate( 'Confirmation' ) }</h1>
+					<p>
+						{ this.props.translate(
+							'Do you want to disconnect Recurring Payments from your Stripe account?'
+						) }
+					</p>
+					<Notice
+						text={ this.props.translate(
+							'Once you disconnect Recurring Payments from Stripe, new subscribers won’t be able to sign up and existing subscriptions will stop working.'
```

Comment:
Let's add a text in bold as well
"Disconnecting your Stripe account here will remove it from all your WordPress.com and Jetpack sites"